services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/NeutronRouterServiceImpl.java

```diff
@@ -98,7 +98,7 @@ public NeutronRouterWebRequestObject saveRouterAndRouterExtraAttribute(NeutronRo
             String routeTableId = UUID.randomUUID().toString();
             routeTable.setId(routeTableId);
             routeTable.setRouteEntities(routeEntities);
-            routeTable.setRouteTableType(RouteTableType.NEUTRON);
+            routeTable.setRouteTableType(RouteTableType.NEUTRON_ROUTER.getRouteTableType());
```

Comment:
Why do we need getRouteTableType()

Reply:
Because I changed the RouteTableType field class type